kfs-core/src/main/resources/edu/arizona/kfs/pdp/spring-pdp.xml

```diff
@@ -18,6 +11,32 @@
         </property>
     </bean>
 
+    <bean id="paymentFileValidationService" parent="paymentFileValidationService-parentBean"/>
+    <bean id="paymentFileValidationService-parentBean" class="edu.arizona.kfs.pdp.service.impl.PaymentFileValidationServiceImpl" abstract="true">
+        <property name="customerProfileService" ref="pdpCustomerProfileService"/>
+        <property name="paymentFileLoadDao" ref="pdpPaymentFileLoadDao"/>
+        <property name="parameterService" ref="parameterService"/>
+        <property name="configurationService" ref="kualiConfigurationService"/>
+        <property name="dateTimeService" ref="dateTimeService"/>
+        <property name="accountService" ref="accountService"/>
+        <property name="subAccountService" ref="subAccountService"/>
+        <property name="objectCodeService" ref="objectCodeService"/>
+        <property name="subObjectCodeService" ref="subObjectCodeService"/>
+        <property name="businessObjectService" ref="businessObjectService"/>
+        <property name="bankService" ref="bankService"/>
+        <property name="originationCodeService" ref="originationCodeService"/>
+        <property name="documentTypeService" ref="rice.kew.documentTypeService"/>
+        <property name="dataDictionaryService" ref="dataDictionaryService"/>
+        <property name="globalTransactionEditService" ref="globalTransactionEditService" />
+    </bean>
+
+    <bean id="globalTransactionEditService" parent="globalTransactionEditService-parentBean" />

```

Comment:
globalTransactionEditService is wired/defined in spring-gl.xml and both gl and pdp are in kfs-core, so no need to define globalTransactionEditService again here in spring-pdp.xml.